create/create-graph.js

```diff
@@ -0,0 +1,30 @@
+const jsc8 = require("jsc8");
+const dotenv = require("dotenv");
+const path = require("path");
+
+const envPath = path.join(__dirname, "..", ".env");
+dotenv.config({ path: envPath });
+
+client = new jsc8({
+  url: process.env.BASE_URL,
+  apiKey: process.env.API_KEY,
+  fabricName: process.env.FABRIC,
+});
+
+// Please be sure that edge collections and vertices have been created before running this script.
```

Comment:
Remove the "please" in all of these. You aren't making a request. You should be writing in the second-person imperative: Be sure that....

Reply:
done done